AC-Music.py

```diff
@@ -1,81 +1,10 @@
+import os
 import datetime
 from playsound import playsound
-currentDT = datetime.datetime.now()
-time_hr = currentDT.strftime("%I %p")
+
 directory = input('Folder Directory ')
 while True:
         currentDT = datetime.datetime.now()
         time_hr = currentDT.strftime("%I %p")
-        mp3_to_play = directory + '/' + time_hr + '.mp3'
-        if time_hr == '12 AM' :
```

Comment:
I lifted `playsound(mp3_to_play)` outside of the `if` blocks because all your `if` blocks have the same code.

The reason why I didn't copy `time_hr` again is because it's already set in the loop.